### DISCUSSION TITLE
Help: How to Set Up Podcastfy’s REST API Locally for External Calls?

### DISCUSSION THREAD

**Discussion by kuhnster01** [souzatharsis/podcastfy#178] [General]
Hi Tharsis,

I’m excited to work with Podcastfy’s REST API but need some help getting it up and running on my own machine. My goal is to host it locally so that an external service, like Make.com, can call it.

Here’s what I’m looking for:
	1.	**API Setup:**
	•	What’s the easiest way to set up the API locally?
	•	Are there any key steps or dependencies I need to be aware of?
	2.	**Hosting Locally:**
	•	How do I expose the local instance for external access (e.g., using ngrok or similar tools)?
	3.	**External Service Integration:**
	•	Are there any configurations needed to ensure external services (like Make.com) can call the API successfully?
	•	Any tips on handling authentication or CORS policies?

I’d really appreciate any guidance or documentation you can share. Thanks in advance for your help!

Best,

Rod

**Comment by souzatharsis** [comment]
Hi Rod,

Thanks for your interest!

1. Please see [quickstart](https://github.com/souzatharsis/podcastfy?tab=readme-ov-file#quickstart-) for setup / dependencies instructions
2. You need an Application Server such as fly.io or DigitalOcean
3. It depends on the service. I am not familiar with Make.com in particular

**Comment by tolnai** [comment]
Hey, I was thinking about the same, wanted to host for my own, locally. I'm a developer, but not working with python so I'm really sure this is not perfect, but it works as a PoC for me: https://github.com/souzatharsis/podcastfy/compare/main...tolnai:podcastfy:server

Would anyone be interested to polish it so that we could also create a PR?
